modules/remove.mjs

```diff
@@ -0,0 +1,12 @@
+const removeBook = (el) => {
+  if (el.classList.contains('delete')) {
+    el.parentElement.remove();
+  }
+};
```

Comment:
- [ ] kindly update the data in local storage after removing it because when refresh the removed data render again.